src/content/components/TreeView.js

```diff
@@ -67,7 +69,7 @@ class TreeViewRowFixedColumns extends Component {
     const { node, columns, index, selected, highlightString } = this.props;
     const evenOddClassName = (index % 2) === 0 ? 'even' : 'odd';
     return (
-      <div className={`treeViewRow treeViewRowFixedColumns ${evenOddClassName} ${selected ? 'selected' : ''}`} style={{height: '16px'}} onClick={this._onClick}>
+      <div className={`treeViewRow treeViewRowFixedColumns ${evenOddClassName} ${selected ? 'selected' : ''}`} style={{height: '16px'}} onMouseDownCapture={this._onClick}>
```

Comment:
Why did you choose onMouseDownCapture instead of onMouseDown? ContextMenuTrigger seems to use onContextMenu, which should fire after the mouse down event.